polaris-react/src/components/Filters/components/FilterPill/tests/FilterPill.test.tsx

```diff
@@ -83,18 +85,22 @@ describe('<Filters />', () => {
     });
 
     it('when not selected, renders a caret icon', () => {
```

Comment:
```suggestion
    it('when not selected, renders a chevron icon', () => {
```

Not sure if this update is worth it but thought I'd still mention it.